docs/pkg_deb.md

```diff
@@ -38,6 +38,7 @@ pkg_deb_impl(<a href="#pkg_deb_impl-name">name</a>, <a href="#pkg_deb_impl-archi
 | distribution |  TBD   | String | optional | "unstable" |
 | enhances |  See http://www.debian.org/doc/debian-policy/ch-relationships.html#s-binarydeps.   | List of strings | optional | [] |
 | homepage |  The homepage of the project.   | String | optional | "" |
+| license |  The license of the project.   | String | optional | "" |
```

Comment:
Don't bother. This is not the real doc file. I think this was an accidentally add while fixing the documentation generation.

Reply:
ok, reverted